packages/repository-typeorm/test/acceptance/typeorm-repository.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/repository-typeorm
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {TypeORMDataSource, TypeORMRepository} from '../..';
+import {MysqlConnectionOptions} from 'typeorm/driver/mysql/MysqlConnectionOptions';
+
+import {Entity as Base} from '@loopback/repository';
+import {Entity, Column, PrimaryGeneratedColumn, Repository} from 'typeorm';
```

Comment:
Interesting. So you are not leveraging decorators from `@loopback/repository` to define models. I think that makes `@loopback/repository` less useful - for example client SDK generators cannot rely on a single set of decorators describing model schemas. Not a big deal, just something to keep in mind and document. I think this also means that all juggler-related stuff should be moved out of `@loopback/repository` into a standalone package, e.g. `@loopback/repository-juggler` so that people wishing to use their own ORM don't have to install juggler dependency.

Reply:
As we discussed online, if we want a single `Repository` instance for consumers (REST API, GraphQL API, etc.), then we need also a single model schema. To get that, I think we need to map our decorators to TypeORM decorators, or provide a method for obtaining LDL definition for a TypeORM model.

Reply:
@bajtos IMO I would like to have a 100% typeorm model, therefore I would expect to decorate using typeorm decorators.

As a developer I would like to have a pure typeorm implementation, not being wrapped or decorated by loopback